### PR TITLE
Update package-fields.yml

### PR DESCRIPTION
**Pull request by alakahakai**
Add fields.yml and update content of fileds.yml and package-fields.yml for netflow integration.

### CODE REVIEW

**Comment by alakahakai** [comment]
Sorry for missing this. fixed now.

File: packages/netflow/dataset/log/fields/package-fields.yml [hunk 1]
```diff
@@ -1,1378 +1,2 @@
 name: netflow
```
Comment (ruflin) [inline packages/netflow/dataset/log/fields/package-fields.yml:1]: I would assume this is still not valid? But CI passes?

Comment (mtojek) [reply packages/netflow/dataset/log/fields/package-fields.yml:1]: Fix: https://github.com/elastic/integrations/pull/44

Comment (mtojek) [reply packages/netflow/dataset/log/fields/package-fields.yml:1]: @ruflin Hm.. not sure if we should wait for the fix here pr push a new separate PR with a proper fix.

Comment (alakahakai) [reply packages/netflow/dataset/log/fields/package-fields.yml]: Sorry, not sure what happened here. I thought I clearly made it an array. I should have reviewed it twice.

**Comment by alakahakai** [comment]
The fields for an integration package are divided into the following three files:

- ecs.yml: ECS compliant fields that are used by this particular dataset.
- package-fields.yml: Package level fields that are used by this particular dataset, which does not exist under <integration-package-name>.<dataset-name>.
- fields.yml: Dataset level fields that are specific to this particular dataset, and non ECS compliant.

**Pull request merged**